docs/fsharp/language-reference/active-patterns.md

```diff
@@ -18,7 +18,7 @@ let (|identifier|) [arguments] valueToMatch = expression
 let (|identifier1|identifier2|...|) valueToMatch = expression
 
 // Partial active pattern definition.
-// Uses a FSharp.Core.option<_> to represent if the type is satisfied at the call site.
+// Can use either FSharp.Core.option<_> or bool to represent if the type is satisfied at the call site.
```

Comment:
Shouldnt't it be : option<>, voption<> or bool ?

Reply:
Yep thanks, didn't update it here, fixed.